src/main/java/model/Product.java

```diff
@@ -1,4 +1,40 @@
 package model;
 
+import java.math.BigDecimal;
+
 public class Product {
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private boolean isAvailable;
+    private ProductGroup group;
+
+    public Product(String name, String description, BigDecimal price, boolean isAvailable, ProductGroup group) {
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.isAvailable = isAvailable;
+        this.group = group;
+    }
+
+    @Override
+    public String toString() {
+        return "Product{" +
+                "name='" + name + '\'' +
+                ", description='" + description + '\'' +
+                ", price=" + price +
+                ", isAvailable=" + isAvailable +
+                ", group='" + group + '\'' +
+                '}';
+    }
+
+    public String getName() {
+        return name;
+    }
+
+
+    public enum ProductGroup {
```

Comment:
why inner class?

Reply:
Make as separate enum. I was thinking that it's part of Product that's why put it inside Product class.